FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/demo_config.h

```diff
@@ -261,6 +261,20 @@ extern void vLoggingPrintf( const char * pcFormatString,
  */
 #define democonfigESTABLISHED_CONNECTIONS_ARRAY_SIZE     10
 
+/**
+ * @brief Size of the custom metrics array.
```

Comment:
More description about the significance of this config will be valuable for the reader.
This could be something like "Number of custom metrics to send to the AWS IoT Device Defender service from the demo".

If the number of custom metrics is fixed within the demo, should this be a configurable value?